<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgGame</class>
 <widget class="QWidget" name="wdgGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="wdgUserPanel" name="panel1" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>378</width>
           <height>125</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="wdgUserPanel" name="panel2" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>378</width>
           <height>125</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="wdgUserPanel" name="panel3" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>378</width>
           <height>125</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="wdgUserPanel" name="panel4" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>378</width>
           <height>125</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTabWidget" name="tab">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="elideMode">
          <enum>Qt::ElideMiddle</enum>
         </property>
         <widget class="QWidget" name="tabGame">
          <attribute name="icon">
           <iconset resource="../images/glparchis.qrc">
            <normaloff>:/glparchis/keko.png</normaloff>:/glparchis/keko.png</iconset>
          </attribute>
          <attribute name="title">
           <string>Juego</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="wdgOGL" name="ogl" native="true">
             <property name="sizeIncrement">
              <size>
               <width>10</width>
               <height>10</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabStatistics">
          <attribute name="icon">
           <iconset resource="../images/glparchis.qrc">
            <normaloff>:/glparchis/cube5.png</normaloff>:/glparchis/cube5.png</iconset>
          </attribute>
          <attribute name="title">
           <string>Estadísticas</string>
          </attribute>
          <layout class="QHBoxLayout">
           <item>
            <widget class="QTableStatistics" name="table">
             <row>
              <property name="text">
               <string>Número de tiradas</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string/>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Dado saca 1</string>
              </property>
              <property name="icon">
               <iconset resource="../images/glparchis.qrc">
                <normaloff>:/glparchis/cube1.png</normaloff>:/glparchis/cube1.png</iconset>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Dado saca 2</string>
              </property>
              <property name="icon">
               <iconset resource="../images/glparchis.qrc">
                <normaloff>:/glparchis/cube2.png</normaloff>:/glparchis/cube2.png</iconset>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Dado saca 3</string>
              </property>
              <property name="icon">
               <iconset resource="../images/glparchis.qrc">
                <normaloff>:/glparchis/cube3.png</normaloff>:/glparchis/cube3.png</iconset>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Dado saca 4</string>
              </property>
              <property name="icon">
               <iconset resource="../images/glparchis.qrc">
                <normaloff>:/glparchis/cube4.png</normaloff>:/glparchis/cube4.png</iconset>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Dado saca 5</string>
              </property>
              <property name="icon">
               <iconset resource="../images/glparchis.qrc">
                <normaloff>:/glparchis/cube5.png</normaloff>:/glparchis/cube5.png</iconset>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Dado saca 6</string>
              </property>
              <property name="icon">
               <iconset resource="../images/glparchis.qrc">
                <normaloff>:/glparchis/cube6.png</normaloff>:/glparchis/cube6.png</iconset>
              </property>
             </row>
             <row>
              <property name="text">
               <string/>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Fichas comidas por mi</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Fichas comidas por otro</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string/>
              </property>
             </row>
             <row>
              <property name="text">
               <string>3 seises seguidos</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string/>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Puntuación</string>
              </property>
             </row>
             <column>
              <property name="text">
               <string>Amarillo</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
              <property name="icon">
               <iconset resource="../images/glparchis.qrc">
                <normaloff>:/glparchis/fichaamarilla.png</normaloff>:/glparchis/fichaamarilla.png</iconset>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Azul</string>
              </property>
              <property name="icon">
               <iconset resource="../images/glparchis.qrc">
                <normaloff>:/glparchis/fichaazul.png</normaloff>:/glparchis/fichaazul.png</iconset>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Rojo</string>
              </property>
              <property name="icon">
               <iconset resource="../images/glparchis.qrc">
                <normaloff>:/glparchis/ficharoja.png</normaloff>:/glparchis/ficharoja.png</iconset>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Verde</string>
              </property>
              <property name="icon">
               <iconset resource="../images/glparchis.qrc">
                <normaloff>:/glparchis/fichaverde.png</normaloff>:/glparchis/fichaverde.png</iconset>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Todos</string>
              </property>
              <property name="icon">
               <iconset resource="../images/glparchis.qrc">
                <normaloff>:/glparchis/star.png</normaloff>:/glparchis/star.png</iconset>
              </property>
             </column>
             <item row="0" column="0">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="0" column="1">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="0" column="2">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="0" column="3">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="0" column="4">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="2" column="0">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="2" column="1">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="2" column="2">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="2" column="3">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="2" column="4">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="3" column="0">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="3" column="1">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="3" column="2">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="3" column="3">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="3" column="4">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="4" column="0">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="4" column="1">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="4" column="2">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="4" column="3">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="4" column="4">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="5" column="0">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="5" column="1">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="5" column="2">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="5" column="3">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="5" column="4">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="6" column="0">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="6" column="1">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="6" column="2">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="6" column="3">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="6" column="4">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="7" column="0">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="7" column="1">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="7" column="2">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="7" column="3">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="7" column="4">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="9" column="0">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="9" column="1">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="9" column="2">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="9" column="3">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="9" column="4">
              <property name="text">
               <string/>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
              <property name="background">
               <brush brushstyle="BDiagPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </property>
             </item>
             <item row="10" column="0">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="10" column="1">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="10" column="2">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="10" column="3">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="10" column="4">
              <property name="text">
               <string/>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
              <property name="background">
               <brush brushstyle="BDiagPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </property>
             </item>
             <item row="12" column="0">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="12" column="1">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="12" column="2">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="12" column="3">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="12" column="4">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="14" column="0">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="14" column="1">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="14" column="2">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="14" column="3">
              <property name="text">
               <string>0</string>
              </property>
              <property name="textAlignment">
               <set>AlignRight|AlignVCenter</set>
              </property>
             </item>
             <item row="14" column="4">
              <property name="background">
               <brush brushstyle="BDiagPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cmdTirarDado">
         <property name="text">
          <string>Tira el dado</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wdgUserPanel</class>
   <extends>QWidget</extends>
   <header location="global">wdgUserPanel</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QTableStatistics</class>
   <extends>QTableWidget</extends>
   <header location="global">qtablestatistics</header>
  </customwidget>
  <customwidget>
   <class>wdgOGL</class>
   <extends>QWidget</extends>
   <header location="global">wdgOGL</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/glparchis.qrc"/>
 </resources>
 <connections/>
</ui>
